deps/lager_syslog-2.0/src/lager_syslog.app.src

```diff
@@ -3,7 +3,7 @@
 {application, lager_syslog,
  [
   {description, "Syslog backend for Lager"},
-  {vsn, "1.2.2"},
+  {vsn, "2.0.3"},
```

Comment:
I downloaded a zip, didn't check the app file for what version it was. Will update